README.md

```diff
@@ -17,7 +17,7 @@ To get a local copy up and running follow these simple example steps.
 Clone the project by using terminal:
 
 ```
-git clone https://github.com/NasKhalil/Morse-code.git
+git clone https://github.com/ReAliens/enumerable.git
 
 cd Morse-code
 
```

Comment:
- [ ] Kindly remove everything that has to do with `morse-code` and let's make sure we `cd` to this current project and how to run this current project 😉

Reply:
Hi @julie-ify 

Thanks for your time spent reviewing our work, it was a lack of concentration.

Cheers 👋

Reply:
thanks i will implement this now :sweat_smile:

Reply:
You guys rock 🔥